doc/release_notes.qbk

```diff
@@ -27,9 +27,15 @@
 
 * [@https://github.com/boostorg/geometry/pull/1045 1045] Support geographic buffer for (multi)linestrings and (multi)polygons
 
+[*Improvements]
+
+* [@https://github.com/boostorg/geometry/pull/1114 1114] Comparative distance: Add support for dynamic geometries
```

Comment:
Is it a problem that the name of the algorithm is comparable distance, not comparative?